CONTRIBUTING.md

```diff
@@ -0,0 +1,117 @@
+# Contributing to GrimoireLab Tutorial
+
+This tutorial is built using 
+[Markdown syntax](https://guides.github.com/features/mastering-markdown/), 
+[Jekyll](https://jekyllrb.com/), and 
+[pmarsceill/just-the-docs](https://github.com/pmarsceill/just-the-docs) theme.
+
+All the Markdown files and Jekyll configuration files are maintained in the 
+[chaoss/grimoirelab-tutorial](http://github.com/chaoss/grimoirelab-tutorial) 
+GitHub repository.
+
+All Contributions are welcome! You can open 
+[pull requests](https://github.com/chaoss/grimoirelab-tutorial/pulls) to add 
+the related content. Feel free to open 
+[issues](https://github.com/chaoss/grimoirelab-tutorial/issues), to report some 
+bug, to propose some enhancement, to ask for some new content, or just to comment 
+or suggest something. Any feedback is appreciated!
+
+If you are willing to setup the tutorial locally
+
```

Comment:
@vchrombie  I usually use ruby 2.7.3 to bundle exec. The latest version causes some error. It could be limited only to my machine but I think we can mention to use an earlier version of ruby if they face any issues.